en/flight_controller/modalai_fc_v1.md

```diff
@@ -0,0 +1,111 @@
+# ModalAI Flight Core v1
+
+The ModalAI Flight Core is a [Pixhawk Series](../flight_controller/pixhawk_series.md) derived flight controller, made in the USA. The Flight Core can be paired with [ModalAI VOXL](https://docs.modalai.com/voxl-datasheet/) for obstacle avoidance and GPS-denied navigation, or used independently as a standalone flight controller.
+
+![FlightCoreV1](../../assets/flight_controller/modalai/fc_v1/main.jpg)
+
+## Quick Summary
+
+The main hardware documentation can be found [here](https://docs.modalai.com/flight-core-datasheet/).
+
+## PX4 Firwmare and Bootloader
+
+Although supported in PX4 v1.10, a few issues have been addressed since than which are currently being maintained by ModalAI [here](https://github.com/modalai/px4-firmware/tree/modalai-1.10).  These changes are mostly picked up in the master branch after the v1.10 release, with ongoing development for improvments in VIO and VOA being added as needed and then upstreamed.
+
+More information about the firmware can be found [here](https://docs.modalai.com/flight-core-firmware/).
+
+## PX4 Bootloader
+
+This board is supported in the PX4 bootloader starting with v5.0 [here](https://github.com/PX4/Bootloader).
```

Comment:
Most end users don't really understand the bootloader - they plugin to QGC and things magically work. Does the board come with bootloader installed? If so, then we could just add a note that this should work plug'n'play with correct version of QGC.

Reply:
Removed bootloader section (it ships with it loaded) and left 'supported in QGC 3.5.7' section  (assuming the next release is 3.5.7)